src/ray/common/cgroup/BUILD

```diff
@@ -43,15 +43,15 @@ ray_cc_library(
 
 ray_cc_library(
     name = "fake_cgroup_setup",
+    testonly = True,
```

Comment:
why cgroup changes in this PR

Reply:
Inter checks all files gets touched compared with main branch